packages/tooltip/src/Tooltip.ts

```diff
@@ -126,7 +131,7 @@ export class Tooltip extends SpectrumElement {
     public selfManaged = false;
 
     @property({ type: Number })
-    public offset = 0;
+    public offset = this.isMobile.matches ? 5 : 4;
```

Comment:
The Spectrum specs never mean "mobile" and "desktop" in this way. They mean it relative to the `scale` the UI is delivered in. I've reached out to the team about tokens for this before we do anything custom here. If we need to do so, we likely want this in https://github.com/adobe/spectrum-web-components/blob/main/tools/styles/scale-medium.css and https://github.com/adobe/spectrum-web-components/blob/main/tools/styles/scale-large.css

Reply:
OK, I wasn't certain if this was referring to the scale or Mobile in general.
And yes it would make sense to check, if this can be solved by a token or spectrum-var.
Will close this for now.